### PR TITLE
Update GetVehicleNumberPlateText.md

### PR DESCRIPTION
**Pull request by cyberpunkleigh**
Update information regarding 0x20 space character appended

### CODE REVIEW

File: ext/native-decls/GetVehicleNumberPlateText.md [hunk 1]
```diff
@@ -8,8 +8,10 @@ apiset: server
 char* GET_VEHICLE_NUMBER_PLATE_TEXT(Vehicle vehicle);
 ```
 
-
 ## Parameters
 * **vehicle**: 
 
+## Note
+If the number plate is less than 8 character return value will have a space at the end to fill the remaining characters
```
Comment (Korioz) [inline ext/native-decls/GetVehicleNumberPlateText.md:15]: Probably better 🤔 :
`If the number plate text is less than 8 characters the result will be appended with spaces until there is 8 characters in it.`

**Review by Korioz — COMMENTED** [review]
Hi 👋,

It seems you're *note* have a typo.

File: ext/native-decls/GetVehicleNumberPlateText.md (hunk 1 above)
Comment (AvarianKnight) [reply ext/native-decls/GetVehicleNumberPlateText.md:15]: I also think that the notes are supposed to come before the ##Paramaters, and wont take a `##` markup

**Review by AvarianKnight — CHANGES_REQUESTED** [review]

**Comment by blattersturm** [comment]
Invalid syntax, user ignored changes requested.

**Pull request closed without merging**